packages/video_player/video_player_avfoundation/test/avfoundation_video_player_test.dart

```diff
@@ -135,6 +135,15 @@ void main() {
       expect(textureId, 3);
     });
 
+    test('create with incorrect asset', () async {
+      try {
+        await player.create(DataSource(sourceType: DataSourceType.asset));
+        fail('should throw PlatformException');
+      } catch (e) {
+        expect(e, isException);
+      }
+    });
+
     test('create with network', () async {
```

Comment:
is this a valid test?

Reply:
where's the asset path here? (I assume you wanna pass in an invalid url or something?)

Reply:
yes I want to input an invalid asset path, hence I put null. or should I be explicit and add an invalid asset path in String?

Reply:
yeah that'd be easier to read